service/src/main/kotlin/app/cash/backfila/dashboard/SeedDbAction.kt

```diff
@@ -0,0 +1,77 @@
+package app.cash.backfila.dashboard
+
+import app.cash.backfila.api.ConfigureServiceAction
+import app.cash.backfila.api.CreateAndStartBackfillAction
+import app.cash.backfila.protos.service.ConfigureServiceRequest
+import app.cash.backfila.protos.service.CreateAndStartBackfillRequest
+import app.cash.backfila.protos.service.CreateBackfillRequest
+import javax.inject.Inject
+import misk.MiskCaller
+import misk.inject.keyOf
+import misk.scope.ActionScope
+import misk.web.Get
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.mediatype.MediaTypes
+
+class SeedDbAction @Inject constructor(
+  val configureServiceAction: ConfigureServiceAction,
+  val createAndStartBackfillAction: CreateAndStartBackfillAction,
+  val scope: ActionScope,
+) : WebAction {
+
+  @Get("/seed")
+  @ResponseContentType(MediaTypes.APPLICATION_JSON)
+  fun seed(): String {
+    scope.close()
+    createServiceWithBackfills("deep-fryer", listOf("ChickenSandwich"))
+    startBackfill("deep-fryer", "ChickenSandwich")
+
+    for (i in 1..100) {
+      createServiceWithBackfills("deep-fryer-$i", listOf("ChickenSandwich"))
+    }
+
+    return "yo"
+  }
+
+  private fun startBackfill(serviceName: String, backfillName: String) {
+    scope.fakeCaller(service = serviceName) {
+      val response = createAndStartBackfillAction.createAndStartBackfill(
+        CreateAndStartBackfillRequest.Builder()
```

Comment:
Does this work? The backfill itself it likely failing? What are you trying to test with this?

Reply:
My testing was just UI testing filtering by services that have running backfills and this seems to end up with the backfills sitting in "Running" so worked for my testing.   (but probably won't be useful for other more indepth testing)